2021/yearbook.html

```diff
@@ -44,6 +44,12 @@ <h2>2021 Software Development Engineer Bootcamp</h2>
           <figcaption>Natasha</figcaption>
         </figure>
       </a>
+      <a href="https://Nonny177.github.io">
+        <figure>
+          <img src="images/Nonny.jpg" alt="Noncedo" target="_blank" />
```

Comment:
The `target="_blank"` only apply to `a` attribute, see https://www.w3schools.com/tags/att_a_target.asp